app/views/paye/paye_total_income_tax.scala.html

```diff
@@ -0,0 +1,203 @@
+        <tr id="starting-rate-for-savings-row">
+          <td scope="row">@Html(Messages("ats.total_income_tax.table.savings",
+              """<span id="start-rate-for-savings-before">"""+includes.currency_amount(viewModel.startingRateForSavings)+"""</span>""",
+              """<span id="start-rate-for-savings-rate">"""+viewModel.startingRateForSavingsRate+"""</span>"""))</td>
+          <td id="starting-rate-for-savings-amount">@includes.currency_amount(viewModel.startingRateForSavingsAmount)</td>
+        </tr>
+        }
+
+        @if(!viewModel.basicRateIncomeTax.isZero) {
+        <tr id="basic-rate-income-tax-row">
+          <td scope="row">@Html(Messages("ats.total_income_tax.table.basic",
+                """<span id="basic-rate-income-tax-before">"""+includes.currency_amount(viewModel.basicRateIncomeTax)+"""</span>""",
+                """<span id="basic-rate-income-tax-rate">"""+viewModel.basicRateIncomeTaxRate+"""</span>"""))</td>
+          <td id="basic-rate-income-tax-amount">@includes.currency_amount(viewModel.basicRateIncomeTaxAmount)</td>
+        </tr>
+        }
+
+        @if(!viewModel.basicRateIncomeTax.isZero) {
+          @if(!viewModel.higherRateIncomeTax.isZero) {
+        <tr id="higher-rate-income-tax-row">
+            <td scope="row">@Html(Messages("ats.total_income_tax.table.higher",
+                """<span id="higher-rate-income-tax-before">"""+includes.currency_amount(viewModel.higherRateIncomeTax)+"""</span>""",
+                """<span id="higher-rate-income-tax-rate">"""+viewModel.higherRateIncomeTaxRate+"""</span>"""))</td>
+          <td id="higher-rate-income-tax-amount">@includes.currency_amount(viewModel.higherRateIncomeTaxAmount)</td>
+        </tr>
+          }
+
+        @if(!viewModel.additionalRateIncomeTax.isZero) {
+        <tr id="additional-rate-income-tax-row">
+          <td scope="row">@Html(Messages("ats.total_income_tax.table.additional",
+              """<span id="additional-rate-income-tax-before">"""+includes.currency_amount(viewModel.additionalRateIncomeTax)+"""</span>""",
+              """<span id="additional-rate-income-tax-rate">"""+viewModel.additionalRateIncomeTaxRate+"""</span>"""))</td>
+          <td id="additional-rate-income-tax-amount">@includes.currency_amount(viewModel.additionalRateIncomeTaxAmount)</td>
+        </tr>
+          }
+        }
+
+      </tbody>
+    </table>
+    }
+
+    @total_income_tax_includes.scottish_table(viewModel.scottishTax, viewModel.scottishRates)
+
+    @total_income_tax_includes.savings_table(viewModel.savingsTax, viewModel.savingsRates)
+
+    @if(viewModel.showDividendsTable) {
+    <table id="dividends">
+      <thead>
+        <tr id="dividends-section-row">
+          <th scope="col">@Html(Messages("ats.total_income_tax.dividends"))</th>
+          <th scope="col">@Html(Messages("generic.amount_pounds"))</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        @if(!viewModel.ordinaryRate.isZero) {
```

Comment:
Hi Steven, this page is a duplicate of this page https://github.com/hmrc/tax-summaries-frontend/blob/master/app/views/total_income_tax.scala.html with relevant paye changes. You have the same code in SA frontend here https://github.com/hmrc/tax-summaries-frontend/blob/master/app/views/total_income_tax.scala.html#L115 it was left as it's non breaking and consistent with what's live. Of course we can update all of these to `viewModel.ordinaryRate.nonZero` which I agree is more readable. But I was under the impression we needed to keep this refactor/separation simple, small, quick and clean?? @chrisjameswright which would you prefer, update these if's in a new commit for this PR or leave them be, so we can continue with the refactor and merge?

Reply:
@chrisjameswright Whichever way you'd like this is fine by me, @scott-d-burgess has asked for this to be added as a new category and made clear here please - https://confluence.tools.tax.service.gov.uk/display/PATS/Technical+work+scope

Reply:
In this case, I think it would be trivial to add to the view model (if it isn't already there) so to aid clarity in the new code it would be a good idea to change - so, it isn't your responsibility to change in existing code, unless you also need to change that for another reason.

I won't be available to add that to the doc today but please feel free to draft and agree between yourselves, and I will review when I can.